### PR TITLE
change importlib to imp.load_module

### PR DESCRIPTION
**Pull request by GuyMoses**

### CODE REVIEW

File: src/lumigo_tracer/auto_instrument_handler.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import imp
```
Comment (saartochner) [inline src/lumigo_tracer/auto_instrument_handler.py:1]: you need to catch the warning and suppress it (just like was did)

File: src/lumigo_tracer/test_module/test.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def cde(*args, **kwargs):
```
Comment (saartochner) [inline src/lumigo_tracer/test_module/test.py:1]: remove

Comment (doriaviram) [reply src/lumigo_tracer/test_module/test.py:1]: He using that on 
```monkeypatch.setenv(ORIGINAL_HANDLER_KEY, "lumigo_tracer/test_module/test.cde")```

Comment (saartochner) [inline src/lumigo_tracer/test_module/test.py:1]: chnge the name to `handler` 😂

**Review by saartochner — APPROVED** [review]
Great work!

**Pull request merged**